example/TrueSDKSample/QATitleSelectionTableViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QATitleSelectionTableViewController.swift
+//  TrueSDKSample
+//
+//  Created by Sreedeepkesav M S on 02/01/20.
+//  Copyright © 2020 Aleksandar Mihailovski. All rights reserved.
+//
+
+import UIKit
+import TrueSDK
+
+extension Notification.Name {
+    static let titleChanged = Notification.Name("loginTitleChanged")
+}
+
+private var titles : [String] {
+    return ["Default",
+            "Login to",
+            "Sign up with",
+            "Sign in to",
+            "Verify with",
+            "Get started with",
+            "Register with"]
+}
+
+class QATitleSelectionTableViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return titles.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = UITableViewCell()
+        cell.textLabel?.text = titles[indexPath.row]
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let title = titleType(for: indexPath.row)
+        postNotificationForTitleChange(with: title, titleName: titles[indexPath.row])
+    }
+}
+
+extension QATitleSelectionTableViewController {
```

Comment:
why extension?

Reply:
Thought of moving the notification part as a separate extension. Just for clarity